src/SDK/Client/Platform/methods/contracts/broadcast.ts

```diff
@@ -5,5 +5,26 @@
  * @param contract - contract
  * @param identity - identity
  */
-import broadcast from "../broadcastRecords";
-export default broadcast;
+import {Platform} from "../../Platform";
+
+export default async function broadcast(this: Platform, dataContract: [any], identity: any): Promise<any> {
+    const { account, client, dpp} = this;
+
+    const dataContractCreateTransition = dpp.dataContract.createStateTransition(dataContract);
+
+    // @ts-ignore
+    const { privateKey } = account.getIdentityHDKey(0);
+
+    dataContractCreateTransition.sign(
+        identity.getPublicKeyById(0),
+        privateKey,
+    );
+
+    const result = await this.dpp.stateTransition.validateStructure(dataContractCreateTransition);
```

Comment:
Is `this` necessary? There is a mismatch between this and the validation in documents/broadcast.
```suggestion
    const result = await dpp.stateTransition.validateStructure(dataContractCreateTransition);
```